openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,103 @@ The following subsections illustrate some of the authorization flows supported b
 
 ## Authorization Code Flow {#authorization-code-flow}
 
-Below is a diagram of a Credential issuance using the Authorization Code Flow, using grant type `authorization_code` as defined in [@!RFC6749].
+The Authorization Code Flow uses the grant type `authorization_code` as defined in [@!RFC6749] to issue Access Tokens.
 
-The diagram shows how a Wallet-initiated flow use case as described in (#use-case-1) is implemented with the Credential Issuance API defined in this specification. Note that the diagram does not illustrate all the optional features of this specification.
+Figure 1 shows the Authorization Code Flows with the two variations that can be implemented for the issuance of the Credentials, as outlined in this specification:
+* **Wallet-initiated variation**, described in (#use-case-4) or (#use-case-6);
```

Comment:
That list does not format correctly in my local html build... Also, seems like other lists do not use bold style for the items, e.g. see lines 169-172

Reply:
fixed
@paulbastian please build every PR locally in the future to check formatting

Reply:
thanks